### PR TITLE
Yunsi/gp 21/connect backend with linebot

### PR DESCRIPTION
**Pull request by yun-si**
I have added posgresql DB testing api.
Besides, I add the pgAdmin4 setting and startup steps into README.md

### Below is the api testing steps.

1. Switch to this branch and run `git pull`
2. Run `docker-compose up -d`
3. Start your Postman
4. GET  `http://localhost:7877/test_create/`  
    See content below means you made it.
    ```{  "message": "success to create table"  }```
5. You can still test `http://localhost:7877/test_select/` or `http://localhost:7877/test_drop/` after step 4
    **test_select** show the content of the table created in step4  
    **test_drop** just drop the table

✨**Remember to drop the table** after testing or other may encounter some trouble when testing.
✨After creating table in step 4, you can also use pgAdmin4 to see the data.

### CODE REVIEW

**Comment by frafishen** [comment]
During your testing procedures, it appears that the Docker image does not rebuild. Instead, you simply use `docker-compose up -d` to recreate the container. Could you explain the rationale behind these testing steps? Additionally, it seems the server is not operational, as HTTP requests from Postman are failing to execute.

File: src/db/pg_api.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+from fastapi import APIRouter, HTTPException
+
+router = APIRouter()
+
+
+@router.get("/access_db/")
+async def access_db():
+    try:
+        # 連接到PostgreSQL
+        conn = psycopg2.connect(
```
Comment (frafishen) [inline src/db/pg_api.py:11]: **(Action Required)** Here is the API layer(routing layer), and we should only process the HTTP request and routing in this layer.
Processing the DB connection here also would let the function process too many things.
You can try to move the database testing things into the `seeds` folder, and let the testing seed function be imported by the routing functions.

**You can adjust the code in this ticket(close the ticket I provided below), or adjust the code in the ticket below**

https://frafishen.atlassian.net/browse/GP-45

image

**Review by frafishen — CHANGES_REQUESTED** [review]

**Comment by yun-si** [comment]
Because I didn't edit the docker-compose.yml and DockerFile, the image does not need to be rebuild. Just start the container. But if your docker haven't been rebuild after I edit the port number, you need to rebuild it.

> During your testing procedures, it appears that the Docker image does not rebuild. Instead, you simply use `docker-compose up -d` to recreate the container. Could you explain the rationale behind these testing steps? Additionally, it seems the server is not operational, as HTTP requests from Postman are failing to execute.

**Comment by frafishen** [comment]
> Because I didn't edit the docker-compose.yml and DockerFile, the image does not need to be rebuild. Just start the container. But if your docker haven't been rebuild after I edit the port number, you need to rebuild it.
> 
> > During your testing procedures, it appears that the Docker image does not rebuild. Instead, you simply use `docker-compose up -d` to recreate the container. Could you explain the rationale behind these testing steps? Additionally, it seems the server is not operational, as HTTP requests from Postman are failing to execute.

I th

File: src/db/pg_api.py (hunk 1 above)
Comment (yun-si) [reply src/db/pg_api.py:11]: I would adjust the code in the ticket below
https://frafishen.atlassian.net/browse/GP-45

**Pull request merged**